modules/vertx-swagger-router/src/main/java/io/vertx/ext/swagger/router/SwaggerRouter.java

```diff
@@ -29,6 +29,7 @@
 
     private static Logger VERTX_LOGGER = LoggerFactory.getLogger(SwaggerRouter.class);
 
+    private static Pattern PATH_PARAMETER_NAME = Pattern.compile("\\{(\\p{Alpha}[\\p{Alpha}\\d]*)\\}");
```

Comment:
Actually this is what the regex used by Vert.X looks like:

```
Matcher m =  Pattern.compile(":([A-Za-z][A-Za-z0-9_]*)").matcher(path);
```

So it has to start with a letter. Maybe I should use a similar one. The syntax I used is confusing and looks like only used in Java.